aproximation_rules.py

```diff
@@ -0,0 +1,58 @@
+import numpy
+
+def mid_rule(n, interval, my_func):
+    '''Approximates the area between a given curve f(x) and the line x=0 within a closed interval [a,b] using the Midpiont Rule '''

```

Comment:
consider using the Google style of docstrings:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
with arguments and returns on the docstrings